src/map_info.h

```diff
@@ -56,10 +56,34 @@ struct LinearHistogramArgs {
   }
 };
 
+struct TSeriesArgs {
+  long interval_ns = -1;
+  long buckets = -1;
+  SizedType inner_type;
```

Comment:
Can we get rid of `inner_type`? It's only used in the output logic; maybe we can infer this some other way?

Reply:
Doesn't seem easy.